invenio/modules/encoder/daemon.py

```diff
@@ -125,8 +122,8 @@ def process_batch(jobfile_path):
     args.append(jobfile_path)
     return launch_task(args)
 
-def watch_directory(new_job_dir=CFG_BIBENCODE_DAEMON_DIR_NEWJOBS,
-                    old_job_dir=CFG_BIBENCODE_DAEMON_DIR_OLDJOBS):
+def watch_directory(new_job_dir=cfg['CFG_BIBENCODE_DAEMON_DIR_NEWJOBS'],
```

Comment:
this might cause troubles. please use `new_job_dir=None` and later `new_job_dir = new_job_dir or cfg['CFG_BIBENCODE_DAEMON_DIR_NEWJOBS']`